packages/types/plugins/expect.ts

```diff
@@ -0,0 +1,93 @@
+export type ExpectPluginConfig = {
+  /**
+   * @title Output format
+   */
+  outputFormat?: 'pretty' | 'json' | 'prettyError' | 'silent';
+  /**
+   * (Deprecated) Formatter
+   * Please use the `outputFormat` option instead.
+   * @deprecated true
+   * @title Formatter
+   */
+  formatter?: ExpectPluginConfig['outputFormat'];
+  /**
+   * Reports failures from expect plugin as errors
+   * in the test report.
+   * @default false
+   * @title Report failures as errors
+   */
+  reportFailuresAsErrors?: boolean;
```

Comment:
there's a new option that got merged here: https://github.com/artilleryio/artillery/pull/1995/files

called `useOnlyRequestNames`, which still needs to be added to the Docs. So might as well add that here already - I'll warn the author to update the docs soon too

Reply:
Documented ✅ Thanks!